app/src/main/java/org/fossasia/phimpme/gallery/activities/SingleMediaActivity.java

```diff
@@ -408,15 +410,15 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
     }
 
 
-
     private void handleEditorImage(Intent data) {
         String newFilePath = data.getStringExtra(EditImageActivity.EXTRA_OUTPUT);
         boolean isImageEdit = data.getBooleanExtra(EditImageActivity.IMAGE_IS_EDIT, false);
 
-        if (isImageEdit){
+        if (isImageEdit) {
 
-        }else{//Or use the original unedited pictures
-            newFilePath = data.getStringExtra(EditImageActivity.FILE_PATH);;
+        } else {//Or use the original unedited pictures
+            newFilePath = data.getStringExtra(EditImageActivity.FILE_PATH);
+            ;
```

Comment:
Why is an extra semicolon placed here?
Also, try not to refactor code until and unless it is absolutely required.

Reply:
I apologise : (
Will take care next time.